source/publishing_data/07_configuring_visualizations/01_configuring_table_view/table.rst

```diff
@@ -1,22 +1,46 @@
-Configuring the table view
-==========================
+Configuring the Table visualization
+===================================
 
-.. admonition:: Note
-   :class: note
+.. admonition:: Prerequisite
+   :class: important
 
-   The table view is always available on all non-empty datasets.
+   For the Table visualization to be available, the dataset must not be empty.
 
-You can configure which fields are displayed, and in which order. Note that it
-only changes the table visualization; removing a field from the table view will
-not remove it from the dataset itself, and it will still be available in other
-visualizations and if you export the dataset.
 
-.. ifconfig:: language == 'en'
+In the edition interface of a dataset, the Visualizations > Table tab allows to configure the Table visualization displayed in the front office, once the dataset is published on the portal.
 
-    .. image:: images/table__table--en.jpg
-        :alt: Table view
+The Table visualization, when available, cannot be disabled.
 
-.. ifconfig:: language == 'fr'
+.. screenshot
 
-    .. image:: images/table__table--fr.jpg
-        :alt: Vue Table
+To configure the Table visualization, follow the indications from the table below.
+
+.. list-table::
+  :header-rows: 1
+
+  * * Option name
+    * Description
+    * Usage
+  * * Default sort
+    * By default, the Table visualization keeps the order of the records as it is after they been processed. It is possible to define another default sort, by choosing a field and a sort type.
+    * 1. Click on the "Default sort" list box.
+      2. Choose a field from the dataset, on which the sorting will be based.
+      3. Click on the second list box, displayed below the first one.
+      4. Choose a sort type.
+  * * Displayed fields
+    * By default, the Table visualization keeps the order of the fields as it is after they have been processed. It is possible to reorder fields, as well as to hide them.
+
+      .. admonition:: Important
+         :class: important
+
+         Reorganizing and hiding fields from the Table visualization configuration tab only changes the Table visualization displayed in the front office. In contrary to when :ref:`using the Processing tab<orderingfields>`, removing a field from the Table visualization will not remove it from the dataset itself, and it will still be available in other visualizations, as well as if the dataset is exported.
```

Comment:
in contrary to the security filter as well
(the important point being that the table field selection is not a security configuration)